src/mqt/yaqs/core/data_structures/mpo.py

```
@@ -597,6 +601,157 @@ def bose_hubbard(
         assert mpo.check_if_valid_mpo(), "MPO initialized wrong"
         return mpo
 
+    @classmethod
+    def trapped_ions_position_grid(
```

Ruff (0.15.18)

[warning] Too many branches (19 > 12)

(PLR0912)